src/main/java/net/library/controller/BookController.java

```diff
@@ -45,7 +46,7 @@ public ResponseEntity<HttpCidSuccessResponse> addBook(@Valid @RequestBody Book b
 
         service.addBook(book);
```

Comment:
I don't see logic in the services. If you use db you should not use mock

Reply:
+